src/interfaces/StoreData.ts

```diff
@@ -19,14 +23,22 @@ type StoreDataMeta = {
     }
 }
 
-type StoreDataEntity = StoreDataMeta & {
+type VirtualStoreDataMeta = StoreDataMeta & {
+    _meta: {
+        virtual: boolean
+        owningResource: string
+        owningRelation: string
+    }
+}
+
+type StoreDataEntity = (StoreDataMeta | VirtualStoreDataMeta) & {
```

Comment:
Is it a realistic scenario to have a virtual entity? I thought currently only embedded collections might get a virtual link?

Reply:
This was cleanup up together with https://github.com/ecamp/hal-json-vuex/pull/260/commits/ad6ec9dddafa75c5ecc495da109dcb9ea49a64b8